src/pages/home/report/ReportActionsView.js

```diff
@@ -110,6 +102,13 @@ class ReportActionsView extends React.Component {
     }
 
     componentDidUpdate(prevProps) {
+        // When the last action changes, wait three seconds, then record the max action
+        if (Visibility.isVisible()) {
+            if (prevProps.reportID !== this.props.reportID) {
```

Comment:
Can this happen in the `reset()` method that fires when the report changes instead?

Reply:
Yup